perception/util/visualizer/traffic_light_visualization/package.xml

```diff
@@ -7,66 +7,18 @@
   <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
   <license>Apache2</license>
 
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
I think you are using `ament_cmake_auto` in your CMakelist so maybe that will need to be reflected here as well.

Reply:
yeah, fixed now